Introduction.Rmd

```diff
@@ -126,14 +126,57 @@ This is a lot to learn, but don't feel overwhelmed. Start with a minimal subset
 
 ## Acknowledgments {#intro-ack}
 
-TODO: when updating this, cover conscious uncoupling in version 2.0.0 and Jim Hester taking over as maintainer in February 2018.
+```{r eval = FALSE, include = FALSE}
+# code generates fodder for the prose below
+library(desc)
+
+get_contributors <- function(pkg = "devtools") {
+  desc_get_authors(system.file("DESCRIPTION", package = pkg))
+}
+# using packages represented by hex stickers in 2019 usethis useR! talk
+# team agreed that was the "right" set of packages
+get_contributors("devtools")
+get_contributors("usethis")
+get_contributors("testthat")
+get_contributors("pkgload")
+
+get_contributors("rcmdcheck")
+get_contributors("sessioninfo")
+get_contributors("remotes")
+get_contributors("revdepcheck")
+
+get_contributors("pkgbuild")
+get_contributors("roxygen2")
+get_contributors("desc")
+get_contributors("withr")
+```
+
+Since the first edition of R Packages was published, the packages supporting the workflows described here have undergone extensive development. The original trio of devtools, roxygen2, and testthat has expanded to include the packages created by the ["conscious uncoupling" of devtools](#setup-usage). Most of these packages originate with Hadley Wickham (HW), because of their devtools roots. There are many other significant contributors, many of whom now serve as maintainers:
+
+  * devtools: HW, [Winston Chang][winston], [Jim Hester][jim] (maintainer, >= v1.13.5)
+  * usethis: HW, [Jennifer Bryan][jenny] (maintainer >= v1.5.0)
+  * testthat: HW (maintainer)
+  * roxygen2: HW (maintainer), [Peter Danenburg][klutometis], [Manuel Eugster][mjaeugster]
+  * remotes: HW, [Jim Hester][jim] (maintainer), [Gábor Csárdi][gabor], [Winston Chang][winston], [Martin Morgan][mtmorgan], [Dan Tenenbaum][dtenenba]
+  * desc: [Gábor Csárdi][gabor] (maintainer), [Kirill Müller][kirill], [Jim Hester][jim]
+  * pkgbuild: HW, [Jim Hester][jim] (maintainer)
+  * pkgload: HW, [Jim Hester][jim] (maintainer), [Winston Chang][winston]
+  * rcmdcheck: [Gábor Csárdi][gabor] (maintainer)
+  * revdepcheck: HW, [Gábor Csárdi][gabor] (maintainer)
+  * sessioninfo: HW, [Gábor Csárdi][gabor] (maintainer), [Winston Chang][winston], [Robert Flight][rmflight], [Kirill Müller][kirill], [Jim Hester][jim]
+  * withr: HW, [Jim Hester][jim] (maintainer), [Kirill Müller][kirill], [Kevin Ushey][kevin], [Winston Chang][winston], [Richard Cotton][richierocks]
```

Comment:
This is a large set of packages, some/many of which might not get specific mentions. But, based on the prep for my uysethis talk, this is the consensus view of where all the functionality went in the conscious uncoupling of devtools.

Reply:
I think it made sense to mention in that part of my usethis talk (plus it has a cool sticker). But I agree it's only peripherally involved in this book. Removed.